net/net_test.go

```diff
@@ -21,6 +21,185 @@ import (
 	"testing"
 )
 
+func TestParseCIDRs(t *testing.T) {
+	testCases := []struct {
+		cidrs         []string
+		errString     string
+		errorExpected bool
+	}{
+		{
+			cidrs:         []string{},
+			errString:     "should not return an error for an empty slice",
+			errorExpected: false,
```

Comment:
Remove `errorExpected` and just check the returned err is nil?

Reply:
there are cases where i error is ok, testing bad data.

Reply:
can work around that by validating the expected error first though